src/UI/Factory.php

```diff
@@ -93,7 +93,11 @@ public function counter();
 	 * description:
 	 *   purpose: The Image component is used to display images of various sources.
 	 *   composition: An Image is composed of the image and an alternative text for screen readers.
-	 *   effect: Images may be included in interacted components but not interactive on their own.
+	 *   effect: >
+	 *     Images may be included in interacted components. Images may also be interactive on their own. Clicking on
```

Comment:
interactive

Reply:
What do you mean here? The first sentence?

Reply:
Images may be included in **interacted** components.

Stumpled over this wording, just saw that this is not even yours. I think it should be interactive, if you feel like it fix. If not, leave as is, I'll fix afterwards.

Reply:
No problem to change this. I just wanted to be sure, because maybe there was a good reason for this wording at that time.

So now this PR can be merged, right @Amstutz ?